examples/templates/kubernetes/main.tf

```diff
@@ -242,8 +242,7 @@ resource "kubernetes_deployment" "main" {
   }
 
   spec {
-    # replicas = data.coder_workspace.me.start_count
-    replicas = 1
+    replicas = data.coder_workspace.me.start_count
     selector {
       match_labels = {
         "app.kubernetes.io/name" = "coder-workspace"
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 279 to 288]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Also, FWIW, I was using this on a GKE autopilot cluster which doesn't support overprovisioning and my pods only ever got `250m` CPU. I think this should probably be configurable, rather than the only option.<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/coder/coder/12194?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
If we ar using the count on deployment and the deployment gets scaled to 0. Does the count value on the pod matter?

Reply:
Oh I see, I'm dumb. Though thinking about it now, I wonder if it would be better to just scale up and down the replicas rather than delete and recreate the deployment each time. I'll just remove the dead code comment for now.

Reply:
My issue still stands about the overprovisioning issue, though. Would like your thoughts